app/services/rbac.rb

```diff
@@ -35,6 +37,19 @@ def load_user_permissions(identity)
       end
     end
 
+    def supported_operation?(attribute_filter)
+      attribute_filter[:operation] == 'in' && attribute_filter[:key] == 'groups.id' ||
+        attribute_filter[:operation] == 'equal' && attribute_filter[:key] == 'group.id'
+    end
+
+    def operation_value(attribute_filter)
+      if attribute_filter[:operation] == 'in' && attribute_filter[:key] == 'groups.id'
+        attribute_filter[:value] == [nil] ? {} : attribute_filter[:value]
+      elsif attribute_filter[:operation] == 'equal' && attribute_filter[:key] == 'group.id'
+        attribute_filter[:value]
+      end
+    end
+
```

Comment:
Method names sound universal to me, but they are scoped for inventory groups. For now let's keep them, but I would put a comment in front of them about this.